grunt-configs/shell.js

```diff
@@ -24,13 +24,31 @@ module.exports = function(grunt, options) {
                      'static/abtests.json'
         },
 
-        jspmFaciaTool: {
+        jspmInstallFaciaTool: {
             command: 'node ../../node_modules/jspm/jspm.js install',
             options: {
                 execOptions: {
                     cwd: 'facia-tool/public'
                 }
             }
+        },
+
+        jspmInstallStatic: {
+            command: './node_modules/.bin/jspm install',
```

Comment:
Have you setup any Github credentials on TC? The reason I'm asking is because we see occasional failures in our build due to `jspm install` failing with "GitHub rate limit reached.". It's rare enough that I haven't bothered fixing it yet, but (1) you may start seeing it too, and (2) if frontend does that regularly, we may exceed our quota more often.